src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -47,7 +47,7 @@ def _list_bids(self, payload):
             filtered_bids = [{"id": v.id, "price": v.price, "energy": v.energy}
                              for _, v in self.market.get_bids().items()
                              if v.buyer == self.device_name]
-            self.redis.publish(
+            self.redis.publish_json(
```

Comment:
This is not part of the fixing the bug, or is it? 
Is there any advantage in comparison to "publish" ?

Reply:
Unfortunately our Jira version does not allow to create hyperlinks to comments, but have a look at the first comment of [the story](https://gridsingularity.atlassian.net/browse/D3ASIM-1946). It is a separate issue, but since it was mentioned in the same bug I tried to solve it in the context of this PR. 
The advantage is that you do not have to wrap the response with json.dumps, which has to be done when issuing `publish`. `publish_json` handles it automatically by doing it inside the method body. 
Of course this could have been avoided if I had more time to write tests that validated the endpoints. Hopefully other bugs like this will be revealed once https://gridsingularity.atlassian.net/browse/TESTS-4 is implemented.